packages/cdktf/lib/terraform-remote-state.ts

```diff
@@ -40,8 +41,10 @@ export abstract class TerraformRemoteState extends TerraformElement {
     return Token.asAny(this.interpolationForAttribute(output));
   }
 
-  private interpolationForAttribute(terraformAttribute: string): any {
-    return `\${data.terraform_remote_state.${this.friendlyUniqueId}.outputs.${terraformAttribute}}`;
+  private interpolationForAttribute(terraformAttribute: string): Expression {
```

Comment:
Should these type of functions return `Expression` or `any`? This appears to be the only one that is `Expression`; however, using `any` seems likely to cause bugs.

Reply:
I think `Expression` would make sense - or do you see other cases?